### PR TITLE
Deploy Mariadb, Memcached, Redis and Postgres on k8s

### PR DESCRIPTION
**Pull request by timlackhan**
I wrote these 4 yaml files to deploy on k8s cluster and they can run well. All yaml files just meet the minimum need to run, except Mariadb and Memcached with secrets. I want to figure out if other functions should be added in these yaml files.

### CODE REVIEW

File: mariadb/README.md [hunk 1]
```diff
@@ -77,3 +77,30 @@ Extra Build ARGs
...
+   --from-literal= mysql-password = <your-mysql-pwd>
+   ```
+
+3. Apply the YAML template configuraton.
+
+   ```
+   kubectl create -f mariadb-deployment.yaml
+   ```
+
+
```
Comment (qzheng527) [inline mariadb/README.md]: Can you add the description of how to connect to the deployed mariadb? 
Through nodeIP:nodeport?

File: memcached/README.md [hunk 2]
```diff
@@ -34,3 +34,28 @@ Extra Build ARGs
...
+
+1. Review the contents of the template file and edit appropriately for your needs.
+
+2. Apply the YAML template configuraton.
+
+   ```
+   kubectl create -f memcached-deployment.yaml
+   ```
+
+
```
Comment (qzheng527) [inline memcached/README.md]: Can you add the description of how to connect to the deployed memcached server?

File: postgres/postgres-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    app: postgres
+spec:
+  type: NodePort
+  ports:
+   - port: 5432
+     targetPort: 5432
+     nodePort: 30001
+  selector:
+   app: postgres
+
```
Comment (qzheng527) [inline postgres/postgres-deployment.yaml]: Also, the description of how-to connect.

File: redis/README.md [hunk 4]
```diff
@@ -41,3 +41,23 @@ Extra Build ARGs
...
+
+1. Review the contents of the template file and edit appropriately for your needs.
+
+2. Apply the YAML template configuraton.
+
+   ```
+   kubectl create -f redis-deployment.yaml
+   ```
+
+
```
Comment (qzheng527) [inline redis/README.md]: description of how to connect, thanks.

**Comment by bluewish** [comment]
From customer perspective, user may:

1, deploy a very simple service 
2, customize configuration file like /etc/mariadb.conf  or /usr/local/share/..conf

Could you add example/template for above 2 for each microservice?

File: mariadb/README.md (hunk 1 above)
Comment (timlackhan) [reply mariadb/README.md]: Thanks for your recommend and I will add it soon!

**Comment by timlackhan** [comment]
@bluewish Of course, thanks for your recommend! I will add it soon!

**Comment by ZhongbaoShi** [comment]
Submitted by other PR, Close this one.

**Pull request closed without merging**